### PR TITLE
FLAG_COLOR.claim is now green/* (white reserved for flag command API) #398

### PR DESCRIPTION
**Pull request by karlthepagan**
FLAG_COLOR.claim is now GREEN instead of WHITE (which is the dialog default).

This makes way for using flags as temporary commands (see #398).

### CODE REVIEW

**Comment by cyberblast** [comment]
hmm you did a good work here, but I'm unsure if we need the migrate. Its a lot of code. to only migrate a few flags. Sorry. 
I'd suggest to keep it clean and simple instead and migrate that one white flag which someone may have manually...

**Comment by karlthepagan** [comment]
Migration is about 25% of this patch, and  error report is another 25% - I'll trim that 50 LoC.

**Comment by karlthepagan** [comment]
cleaned up and rebased on to dev

File: global.js [hunk 1]
```diff
@@ -54,7 +54,31 @@ var mod = {
...
+                        color: COLOR_GREEN,
+                        secondaryColor: COLOR_RED,
+                        filter: {'color': COLOR_GREEN, 'secondaryColor': COLOR_RED }
+                    },
+                    reserve: { // reserve this room
+                        color: COLOR_GREEN,
+                        secondaryColor: COLOR_GREY,
+                        filter: {'color': COLOR_GREEN, 'secondaryColor': COLOR_GREY },
+                    },
+                    mining: { // from branch:feature/remoteMining
```
Comment (cyberblast) [inline global.js]: oops ;)
 mining: { // from branch:feature/remoteMining

Comment (karlthepagan) [inline global.js]: intentional - but I had 2 helpful choices here
* send a PR into feature/remoteMining
* leave it in with the note that it was reserved for the feature

Comment (karlthepagan) [inline global.js]: resolved

**Comment by whardeman** [comment]
Could the migrate code be rolled up into an ad hoc module to be used when updating?

**Comment by karlthepagan** [comment]
I'll push a migrations branch with this viral: https://github.com/karlthepagan/screeps.ocs.personal/blob/master/viral.flagDir.js

**Pull request merged**